### PR TITLE
RM-132: Release notes 5.6.28-25.14

### PR DESCRIPTION
**Pull request by azhebel**

### CODE REVIEW

File: doc/source/release-notes/Percona-XtraDB-Cluster-5.6.28-25.14.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Percona is glad to announce the release of |Percona XtraDB Cluster| 5.6.28-25.14 on January 29, 2016. Binaries are available from the `downloads section <http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/release-5.6.28-25.14/>`_ or from our :ref:`software repositories <installation>`.
+
+Percona XtraDB Cluster 5.6.28-25.14 is now the current release, based on the following:
+
+* `Percona Server 5.6.28-76.1 <http://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.28-76.1.html>`_
+
+* `Galera Replicator 3.14 <https://github.com/codership/galera/issues?q=milestone%3A25.3.14>`_
+
+* `Codership wsrep API 25.12 <https://github.com/codership/mysql-wsrep/issues?q=milestone%3A5.6.28-25.13>`_
```
Comment (kbauskar) [inline doc/source/release-notes/Percona-XtraDB-Cluster-5.6.28-25.14.rst]: API 25.12 -> API 25.13

File: doc/source/wsrep-system-index.rst [hunk 2]
```diff
@@ -329,16 +329,27 @@ This variable can be used to reject queries for the node. This can be useful dur
...
-   :version 5.6.24-25.11: Variable available both in global ans session scope
+   :version 5.6.24-25.11: Variable available both in global and session scope
    :cli: Yes
    :conf: Yes
    :scope: Session, Global
    :dyn: No
    :default: OFF
 
-This variable controls if *MyISAM* will be replicated or not. *MyISAM* replication is still experimental and that is one of the reasons why this variable is set to ``OFF`` by default. *MyISAM* ``CREATE TABLE`` isn't replicated any more when :variable:`wsrep_replicate_myisam` is ``OFF``.
+This variable defines whether MyISAM should be replicated or not. It is disabled by default, because MyISAM replication is still experimental. MyISAM ``CREATE TABLE`` isn't replicated any more when :variable:`wsrep_replicate_myisam` is disabled.
```
Comment (kbauskar) [inline doc/source/wsrep-system-index.rst]: MyISAM CREATE TABLE isn't replicated ... 
This has changed starting 5.6.27. Please check updated semantics. We replicate DDL but block DML based on wsrep_replicate_myisam configuration.

So the points below contradict this stmt.

File: doc/source/wsrep-system-index.rst [hunk 3]
```diff
@@ -329,16 +329,27 @@ This variable can be used to reject queries for the node. This can be useful dur
...
+
+.. note:: You should keep in mind the following when for MyISAM replication:
+
+  * DDL (CREATE/DROP/TRUNCATE) statements on MyISAM will be replicated irrespective of :variable:`wsrep_replicate_miysam` value
+  * DML (INSERT/UPDATE/DELETE) statements on MyISAM will be replicated only if :variable:`wsrep_replicate_myisam` is enabled
+  * SST will get full transfer irrespective of :variable:`wsrep_replicate_myisam` value (it will get MyISAM tables from donor)
+  * Difference in configuration of ``pxc-cluster`` node on `enforce_storage_engine <https://www.percona.com/doc/percona-server/5.6/management/enforce_engine.html>`_ front may result in picking up different engine for same table on different nodes
+  * ``CREATE TABLE AS SELECT`` (CTAS) statements use non-TOI replication and are replicated only if there is involvement of InnoDB table that needs trx (involvement of MyISAM table will cause CTAS statement to skip replication)
```
Comment (kbauskar) [inline doc/source/wsrep-system-index.rst]: s/trx/transaction/g

**Comment by kbauskar** [comment]
Some small tail end comment other wise looks good.
Hrovje need to do a final review.

**Comment by hrvojem** [comment]
LGTM

**Pull request merged**